### PR TITLE
Added log.Fatal in possible fatal errors returns

### PR DESCRIPTION
**Pull request by ike-cunha**
Related issue number: user/hcunha/001
What change(s) you're making: added log.Fatal in possible fatal errors returns
Why you're making the change(s): to improve error handling
How your change(s) will resolve the why: will handle fatal errors with proper syntax

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -26,24 +28,26 @@ func main() {
 	// Create a new Discord session using the provided bot token.
 	dg, err := discordgo.New("Bot " + Token)
 	if err != nil {
-		fmt.Println("error creating Discord session,", err)
+		log.Fatal("Error creating Discord session", err)
 		return
```
Comment (marcuxyz) [inline main.go:32]: You don't need of `return` statement here. Cause, `log.Fatal` function emit a `os.Exit(1)`

File: main.go [hunk 2]
```diff
@@ -26,24 +28,26 @@ func main() {
...
+	// Identify is sent during initial handshake with the discord gateway.
+	// In this case, will listen to group messages.
 	dg.Identify.Intents = discordgo.IntentsGuildMessages
 
 	// Open a websocket connection to Discord and begin listening.
 	err = dg.Open()
 	if err != nil {
-		fmt.Println("error opening connection,", err)
+		log.Fatal("error opening connection,", err)
 		return
```
Comment (marcuxyz) [inline main.go:47]: You don't need of `return` statement here. Cause, `log.Fatal` function emit a `os.Exit(1)`

File: main.go [hunk 3]
```diff
@@ -26,24 +28,26 @@ func main() {
...
 	}
 
 	// Register the messageCreate func as a callback for MessageCreate events.
+	// Every time a message is created, func messaCreate is fired.
 	dg.AddHandler(messageCreate)
 
-	// In this example, we only care about receiving message events.
+	// Identify is sent during initial handshake with the discord gateway.
+	// In this case, will listen to group messages.
```
Comment (marcuxyz) [inline main.go:40]: those comments can be deleted

File: main.go [hunk 4]
```diff
@@ -58,15 +62,14 @@ func main() {
 func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 
 	// Ignore all messages created by the bot itself
-	// This isn't required in this specific example but it's a good practice.
 	if m.Author.ID == s.State.User.ID {
```
Comment (marcuxyz) [inline main.go:65]: who is `m`?

File: main.go [hunk 5]
```diff
@@ -58,15 +62,14 @@ func main() {
...
 		return
 	}
+
 	// If the message is "ping" reply with "Pong!"
 	toBino := strings.Contains(m.Content, "bino")
 	if toBino {
 		s.ChannelMessageSend(m.ChannelID, "E cilada bino!!!")
```
Comment (marcuxyz) [inline main.go:72]: who is `s`?

**Review by marcuxyz — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (marcuxyz) [reply main.go:32]: Please check: https://golang.org/src/log/log.go?s=10278:10306#L323

File: main.go (hunk 2 above)
Comment (marcuxyz) [reply main.go:47]: See: 
https://golang.org/src/log/log.go?s=10278:10306#L323

File: main.go (hunk 1 above)
Comment (ike-cunha) [reply main.go:32]: Perfect!

File: main.go (hunk 4 above)
Comment (ike-cunha) [reply main.go:65]: `m` stands for Message. I changed the param name to make it simpler.

File: main.go (hunk 2 above)
Comment (ike-cunha) [reply main.go:47]: Ok

File: main.go (hunk 5 above)
Comment (ike-cunha) [reply main.go:67]: `s` stands for Session. I changed the param name to make it simpler.

**Review by marcuxyz — APPROVED** [review]

**Pull request merged**